background.js

```diff
@@ -342,6 +342,13 @@ Service.registerFrame = function(message, sender, sendResponse) {
 Service.unRegisterFrame = function(message, sender, sendResponse) {
     unRegisterFrame(sender.tab.id);
 };
+Service.moveTab = function(message, sender, sendResponse) {
+    var newPos = parseInt(message.position);
+    var activeTabId = sender.tab.id;
+    if (newPos > -1 && newPos < 10) {
+        chrome.tabs.move(activeTabId, {index: newPos});
+    }
+}
```

Comment:
Why newPos must be less than 10?
And please append `;` at line 351 to make sure that js-beautify will help us to beautify the code.